2_Functions_Part_2.ipynb

```diff
@@ -77,9 +77,20 @@
   },
```

Comment:
<p>you can solve this in a more elegant way by iterating over ingredients differently, think how</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week05-sagin132/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='135'/>